app/sessionStore.js

```diff
@@ -233,6 +233,8 @@ module.exports.cleanAppData = (data, isShutdown) => {
   data.temporarySiteSettings = {}
   // Delete Flash state since this is checked on startup
   delete data.flashInitialized
+  // Delete Recovery status on shut down
+  delete data.ui.about.preferences.recoverySucceeded
```

Comment:
plz document in docs/state.md. also isn't this going to throw an error for users who have never enabled the ledger?